guides/get-started-microservices-on-kubernetes/simple-microservice-part1.adoc

```diff
@@ -0,0 +1,252 @@
+= {simple-microservice-part1}
+:summary: ONELINE EXPLANATION OF THE GUIDE
+:includedir: ../_includes
+include::{includedir}/_attributes.adoc[]
+include::./_includes/_titles.adoc[]
+:prerequisites-time: 10
+
+In this guide, you will learn how to create and run a link:https://docs.docker.com/guides/docker-concepts/the-basics/what-is-an-image/[Docker Image] containing a link:https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Service] implemented using WildFly.
+
+include::{includedir}/_prerequisites.adoc[]
+
+* link:/get-started[Getting Started with WildFly]
+
+include::_includes/_constants.adoc[]
+
+== Maven Project
+
+In link:/get-started[Getting Started with WildFly], we created a simple Jakarta EE application Maven Project containing a link:https://jakarta.ee/specifications/restful-ws/[Jakarta RESTful Web Service]: we'll pick up from there;
+
+=== pom.xml
+
+Since we are going to deploy on link:https://kubernetes.io/[Kubernetes], we need to add some features that are necessary when running on link:https://kubernetes.io/[Kubernetes]: link:https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/[Liveness, Readiness and Startup Probes];
+
+We can add these features by adding the link:https://github.com/wildfly-extras/wildfly-cloud-galleon-pack[wildfly-cloud-galleon-pack] to our application; add the following to the the link:https://github.com/wildfly/wildfly-maven-plugin/[wildfly-maven-plugin] configuration in the `pom.xml` file:
+
+[source,xml,subs="normal"]
+----
+    <feature-pack>
+        <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:{version-wildfly-cloud-galleon-pack}</location>
+    </feature-pack>
+----
+
+You should end up with the link:https://github.com/wildfly/wildfly-maven-plugin/[wildfly-maven-plugin] configured like in the following:
+
+[source,xml,subs="normal"]
+----
+    <plugin>
+        <groupId>org.wildfly.plugins</groupId>
+        <artifactId>wildfly-maven-plugin</artifactId>
+        <version>{version-wildfly-maven-plugin}</version>
+        <configuration>
+            <feature-packs>
+                <feature-pack>
+                    <location>org.wildfly:wildfly-galleon-pack:{version-wildfly-galleon-pack}</location>
+                </feature-pack>
+                <feature-pack>
+                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:{version-wildfly-cloud-galleon-pack}</location>
+                </feature-pack>
+            </feature-packs>
+            <layers>
+                <layer>cloud-server</layer>
+            </layers>
+        </configuration>
+        <executions>
+            <execution>
+                <goals>
+                    <goal>package</goal>
+                </goals>
+            </execution>
+        </executions>
+    </plugin>
```

Comment:
Is it possible to use Wildfly Glow here? I find it much more user friedly.

Reply:
I prefer not to use glow, since its usability was under discussion... it might be yet another post in this mini-serie